providers/database_provider.ts

```diff
@@ -80,6 +87,20 @@ export default class DatabaseServiceProvider {
     }
   }
 
+  /**
+   * Register TestUtils database macro
+   */
+  protected async registerTestUtils(db: Database) {
+    if (this.app.getEnvironment() === 'test') {
+      const { TestUtils } = await import('@adonisjs/core/test_utils')
+      const ace = await this.app.container.make('ace')
```

Comment:
not sure here too. should we resolve the Ace binding immediately ?

we could resolve it only when the user calls `testUtils.db().migrate()` for example, but the code would be a bit more complicated. so, i am wondering whether it's worth bothering or not